README.md

```diff
@@ -1,85 +1,87 @@
-<p align="center">
- <a href="https://www.gardenlinux.io/">
-  <img
-     src="https://raw.githubusercontent.com/gardenlinux/gardenlinux/main/logo/gardenlinux-logo-black-text.svg"
-     width="380"
-  />
- </a>
-</p>
-
-<hr />
-<p align="center">&bull;
-    <a href="#Features">Features</a> &bull;
-    <a href="#build-requirements">Build Requirements</a> &bull;
-    <a href="#build-options">Build Options</a> &bull;
-    <a href="#quick-start">Quick Start</a> &bull;
-    <a href="#customize-builds">Customize</a> &bull;
-    <a href="#garden-linux-releases">Releases</a> &bull;
-</p>
-<hr />
-
```

Comment:
Any particular reason we are doing away with the fancy header?

Reply:
We have menu sub items (`Quick Start` chapter), now. Therefore, I switched to a regular ToC. Readding this header would result in duplicated content.